csrc/mmdeploy/codebase/mmaction/format_shape.cpp

```diff
@@ -50,6 +51,7 @@ Result<void> FormatShape::Apply(Value& data) {
   }
 
   Tensor dst;
+  data = Value{};
```

Comment:
why clear `data`?

Reply:
The next node treate data as a object, while in this node, the data is a list.